csrc/kernel_cache.h

```diff
@@ -490,7 +502,9 @@ class FusionExecutorCache {
   //! create new fusion executor cache at a given device to handle kernel
   //! generation of dynamic sizes
   //! fusion executor is taking the ownership of `fusion`
-  explicit FusionExecutorCache(std::unique_ptr<Fusion> fusion);
+  explicit FusionExecutorCache(
+      std::unique_ptr<Fusion> fusion,
+      int64_t fusion_id = 0);
```

Comment:
so `0` here means it's invalid?!

Reply:
A negative id is invalid.  0 is just the default id.

Reply:
We use `FusionExecutorCache` in the cpp tests where we don't care to specify the `fusion_id`.